scripts/make/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+cd ../..
+repo_root=$(pwd)
+
+source "$repo_root/.build.env"
+
+if [ "$drupal_build" == "Y" ]; then
+  docker run -ti -v $repo_root:/var/www/html -w /var/www/html "$drupal_build_php_tag" /bin/bash -c "$drupal_build_command"
```

Comment:
Maybe rename `drupal_build_command` to `drupal_install_command`?

Reply:
tldr; So I think Dan's right (probably) :)

Because this is the point the dependencies are downloaded so it should be instal

Install means something different in Drupal land doesn't it... thats where the db is setup (i.e. the application is actually installed).  

In fact, that is what install usually means in any Makefile used to build an application... (i.e. its always ./configure && make build && make install)

But we are using install in the composer / npm way of thinking of it which means get dependencies... 

well thats, er, interesting... naming things eh?!

Reply:
agreed